src/zoid/card-fields/component.jsx

```diff
@@ -128,9 +129,10 @@ const url = () =>
   `${getPayPalDomain()}${__PAYPAL_CHECKOUT__.__URI__.__CARD_FIELD__}`;
 
 const prerenderTemplate = ({ props, doc }) => {
-  return (
-    <CardPrerender nonce={props.nonce} height={props.style?.height} />
-  ).render(dom({ doc }));
+  const height = props.style?.height || props.style?.input?.height || null;
```

Comment:
One nit pick is to use `??` rather than `||` because the it will only be triggered by `null` or `undefined` values. Merchant could potentially pass `height` as 0 (or any falsy value for that matter) and that would be a valid value but we would be nulling it and applying the default height.

Reply:
This is valid feedback and not a nitpick at all. I'll make the change!